core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -303,7 +304,7 @@ class PartKeyLuceneIndex(ref: DatasetRef,
       if (name.nonEmpty && value.nonEmpty &&
         (always || facetEnabledForLabel(name)) &&
         value.length < FACET_FIELD_MAX_LEN) {
-        facetsConfig.setRequireDimensionDrillDown(name, false)
+        facetsConfig.setDrillDownTermsIndexing(name, DrillDownTermsIndexing.NONE)
```

Comment:
This method is now deprecated, and a replacement is suggested [here](https://lucene.apache.org/core/8_11_2/facet/org/apache/lucene/facet/FacetsConfig.html#setRequireDimensionDrillDown-java.lang.String-boolean-).